docs/user_manual/source/load_store_unit.rst

```diff
@@ -12,31 +12,33 @@ supported.
+  +---------------------------+-----------------+------------------------------------------------------------------------------------------------------------------------------+
+  | ``data_mem_type_o[1:0]``  | output          | Memory Type attributes (cacheable, bufferable), sent together with ``data_req_o``.                                           |
+  +---------------------------+-----------------+------------------------------------------------------------------------------------------------------------------------------+
+  | ``data_prot_o[2:0]``      | output          | Protection attributes, sent together with ``data_req_o``.                                                                    |
+  +---------------------------+-----------------+------------------------------------------------------------------------------------------------------------------------------+
+  | ``data_wdata_o[31:0]``    | output          | Data to be written to memory, sent together with ``data_req_o``.                                                             |
+  +---------------------------+-----------------+------------------------------------------------------------------------------------------------------------------------------+
+  | ``data_we_o``             | output          | Write Enable, high for writes, low for reads. Sent together with ``data_req_o``.                                             |
+  +---------------------------+-----------------+------------------------------------------------------------------------------------------------------------------------------+
+  | ``data_rvalid_i``         | input           | ``data_rvalid_i`` will be high for exactly one cycle to signal the end of the response phase of for both read and write      |
+  |                           |                 | transactions. For a read transaction ``data_rdata_i`` holds valid data when ``data_rvalid_i`` is high.                       |
+  +---------------------------+-----------------+------------------------------------------------------------------------------------------------------------------------------+
+  | ``data_rdata_i[31:0]``    | input           | Data read from memory. Only valid when ``data_rvalid_i`` is high.                                                            |
+  +---------------------------+-----------------+------------------------------------------------------------------------------------------------------------------------------+
+  | ``data_err_i``            | input           | A data interface error occurred. Only valid when ``data_rvalid_i`` is high.                                                  |
+  +---------------------------+-----------------+------------------------------------------------------------------------------------------------------------------------------+
 
```

Comment:
Same as for instruction interface, missing *par, *achk and *rchk?

Reply:
Will add